src/renderer/store/settings/index.ts

```diff
@@ -55,28 +59,43 @@ export const useSettings = create<StoreState & StoreReducers>((set, get) => ({
       })
     );
   },
+  setIsoModTime: (isoModTime) => {
+    set((state) =>
+      produce(state, (draft) => {
+        draft.settings.isoModTime = isoModTime;
+      })
+    );
+  },
   setReplayDirectory: (dir) => {
     set((state) =>
       produce(state, (draft) => {
         draft.settings.rootSlpPath = dir;
       })
     );
   },
-  verifyIsoPath: async (isoPath, shouldSetPath) => {
+  verifyIsoPath: async (isoPath) => {
     // Indicate that we're loading
     set({
       verifyingIso: true,
       validIsoPath: null,
     });
 
     try {
-      const res = await verifyISO(isoPath);
-      set({ validIsoPath: res.valid });
+      const storedModTime = get().settings.isoModTime;
```

Comment:
If you're calling `get()` a lot, something that might be easier and potentially more readable is using object destructuring syntax. e.g

```typescript
const { settings, setIsoPath, setIsoModTime } = get();
```